uint32 frame_id
uint32 num_object
Bbox[] bboxes
